definition.tex

```diff
@@ -1,3 +1,93 @@
-\section{Definitions}
+\section{Preliminaries}
 
 To be written...
+
+\paragraph{Basic notation.}
+\label{sec:notation}
+
+To be written...
+\begin{itemize}[noitemsep]
+    \item security parameters
+    \item probabilistic notation
+\end{itemize}
+
+\subsection{Collision Resistant Hash Function}
+
+To be written...
+
+\newcommand{\getsr}{\gets_{\ms{R}}}
+
+\begin{definition}[Collision Resistance Hash Function]
+    We say that a family of hash function $\calH_\calK = \{ H_k: \calX \to
+    \calY \}_{k \in \calK}$ for a key space $\calK = \calK_{\lambda \in \N}$,
+    domain $\calX = \calX_{\lambda \in \N}$ and range $\calY = \calY_{\lambda
+    \in \N}$ is \emph{collision resistant} if there exists a negligible
+    function $\negl(\lambda)$ such that for all efficient adversaries~$\calA$,
+    we have
+    \[ \Pr\Big[~ k \getsr \calK;~ (x_0, x_1) \gets \calA(\lambda, H_k);~
+    x_0 \neq x_1 \land H_\lambda(x_0) = H_\lambda(x_1) ~\Big] = \negl(\lambda)
+.\]
+\end{definition}
+
+\subsection{Accumulators}
+
+To be written...
+
+\newcommand{\Piacc}{\Pi_\ms{acc}}
+
+\begin{definition}[Accumulator]
+    A \emph{variable-length} accumulator scheme $\Piacc$ for an input domain
+    $\calX_{\lambda, \in \N}$, proof space $\calU_\lambda$, and commitment space
```

Comment:
Just a small typo here probably `,` after `\lambda` on line 40

Reply:
Great, thanks for the catch!